pkg/step/feature/derivation.go

```diff
@@ -133,9 +133,9 @@ func newRowValue(columnTypeList []*sql.ColumnType) ([]interface{}, error) {
 	for idx, ct := range columnTypeList {
 		typeName := ct.DatabaseTypeName()
 		switch unifyDatabaseTypeName(typeName) {
-		case "VARCHAR", "TEXT", "STRING":
+		case "CHAR", "VARCHAR", "TEXT", "STRING":
 			rowData[idx] = new(string)
-		case "INT":
+		case "INT", "TINYINT":
```

Comment:
Will there be some test cases for these data types?

Reply:
Done.